### PR TITLE
Update bettertouchtool to 2.660

### PR DESCRIPTION
**Pull request by core-code**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/bettertouchtool.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
     # bettertouchtool.net/releases was verified as official when first introduced to the cask
     url "https://bettertouchtool.net/releases/btt#{version}.zip"
-    appcast 'https://bettertouchtool.net/releases/'
+    appcast 'https://updates.bettertouchtool.net/highsierra_new.xml?bttversion=967'
```
Comment (vitorgalvao) [inline Casks/bettertouchtool.rb]: Won’t the fixed version at the end be a problem?

**Comment by core-code** [comment]
as i understand it, it will be a problem once they decide to charge all users again for an upgrade again.


but when and if that happens we can change the number?

**Comment by vitorgalvao** [comment]
> once they decide to charge all users for an upgrade again

It’s a subscription now. You pay for two years at a time so there’s no moment where everyone has to pay again. The old `appcast` might still be better.

**Comment by core-code** [comment]
i didn't get the part of the subscription - just refused to pay again ;)

i am OK with dropping the appcast.

alternative could be to change the cask to carry both version numbers as many other casks do

**Pull request closed without merging**